Services/Container/classes/Screen/MemberViewLayoutProvider.php

```diff
@@ -30,39 +30,48 @@ public function isInterestedInContexts() : ContextCollection
         return $this->context_collection->repository();
     }
 
+    /**
+     * return false | ModeInfo;
+     */
+    public static function getMemberViewModeInfo(\ILIAS\DI\Container $dic)
```

Comment:
you could declare the return type here. but this would need some adjustemnts since here is bool|ModeInfo mixed up. maybe returning `null` and declare `?ModeInfo` as return type? or factor out the status of the memberview e.g. to a context-data